build/android/BUILD.gn

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
I think the operation of generating xwalk_core_library/aar operation has nothing to do with the source code and looks like a build-target wrapper of python scripts, that's why I put them here.
Maybe need to find a better place, //xwalk/BUILD.gn?  or //xwalk/runtime/android/BUILD.gn?